client/app/src/main/AndroidManifest.xml

```diff
@@ -16,7 +16,7 @@
         <activity
             android:name="uk.ac.cam.cl.echo.extrusionfinder.client.ImageCaptureActivity"
             android:screenOrientation="portrait"
-            android:theme="@style/Theme.AppCompat.NoActionBar" >
+            android:theme="@android:style/Theme.Holo.NoActionBar.Fullscreen" >
```

Comment:
Removes the status bar working around issues with the camera preview being squashed and so not fitting the overlay properly.